_i18n/en.yml

```diff
@@ -54,36 +54,33 @@ index:
+  whatis: Learn more about Monero
   get_started: Get Started
-  why_monero_is_different: Why Monero is different
-  monero_is_secure: Monero is secure
-  monero_is_secure_para: Monero is a decentralized cryptocurrency, meaning it is secure digital cash operated by a network of users. Transactions are confirmed by distributed consensus and then immutably recorded on the blockchain. Third-parties do not need to be trusted to keep your Monero safe.
-  monero_is_private: Monero is private
-  monero_is_private_para: Monero uses ring signatures, ring confidential transactions, and stealth addresses to obfuscate the origins, amounts, and destinations of all transactions. Monero provides all the benefits of a decentralized cryptocurrency, without any of the typical privacy concessions.
-  monero_is_untraceable: Monero is untraceable
-  monero_is_untraceable_para: Sending and receiving addresses as well as transacted amounts are obfuscated by default. Transactions on the Monero blockchain cannot be linked to a particular user or real-world identity.
-  monero_is_fungible: Monero is fungible
-  monero_is_fungible_para1: Monero is
-  monero_is_fungible_para2: fungible
-  monero_is_fungible_para3: because it is private by default. Units of Monero cannot be blacklisted by vendors or exchanges due to their association in previous transactions.
+  choosewallet: Choose a Wallet
+  choosewallet_para: To use Monero, the first thing you are going to need is a wallet. Visit our Downloads page and get the right wallet for you. The Monero wallets are available for a variety of platforms and contain everything you need to use Monero immediately and, if you want, run your own node.
   downloads: Downloads
-  downloads_windows: Monero for Windows
-  downloads_mac: Monero for Mac
-  downloads_linux: Monero for Linux
-  downloads_blockchain: Latest Blockchain
-  different_system: Need it for a different operating system?
-  view_all_downloads: View all available downloads here.
-  latest_news: Latest News
-  more_news: More news
+  getcoins: Get Some Coins
+  getcoins_para: After you install a wallet, you need to get some Monero. There are multiple ways to acquire some coins to spend, like mining or working in exchange for Monero, but the easiest way is to use an exchange and convert your fiat money into XMR. many exchanges, centralized and decentralized, list Monero (XMR).
+  exchanges: Exchanges
+  useit: "It's a currency: Use it!"
+  useit_para: Monero is a currency and can be exchanged for goods and services, privately and with very low fees. Many entities will gladly accept XMR for payments; take a look at our list of merchants currently accepting Monero.
+  merchants: Merchants
+  answers: Find answers to your questions
+  faq: FAQ
+  faq_para: We've heard a lot of questions over the years and have compiled, for your convenience, a thorough and varied FAQ. Don't worry, if your questions are not on here, you can always ask the community.
```

Comment:
A bit clunky to read. Maybe something more like:

> We have compiled a list of frequently asked questions, and have a large community with members ready to answer any questions you cannot find the answer to.

Reply:
On a second look i find myself not 100% convinced about your suggestion (no problem with changing my version tho). I would avoid repeating both 'questions' and 'answer' in such short sentence.

Reply:
We agreed to just remove `, for your convenience,` from the original sentence.